tests/Fixtures/Symfony/TestAppKernel.php

```diff
@@ -109,6 +110,21 @@ protected function configureContainer(ContainerBuilder $c, LoaderInterface $load
         if ($this->coverageEnabled && 'cov' !== $this->environment) {
             $loader->load($confDir.'/cov/**/*'.self::CONFIG_EXTENSIONS, 'glob');
         }
+
+        $c->addCompilerPass(
```

Comment:
Correct, me if I'm wrong but there is an available special container that allows to access private services in test suites, so we do not need that.

https://symfony.com/blog/new-in-symfony-4-1-simpler-service-testing
https://symfony.com/doc/current/testing.html#accessing-the-container

Reply:
There is, somehow I never noticed the green box mentioning the `test.service_container` id, thanks for pointing this out!